charts/otel-cloud-stack/values.yaml

```diff
@@ -658,12 +658,44 @@ logsCollector:
       k8sattributes:
         passthrough: false
         pod_association:
+          - sources:
+              - from: resource_attribute
+                name: k8s.pod.uid
+          - sources:
+              - from: resource_attribute
+                name: k8s.pod.name
+              - from: resource_attribute
+                name: k8s.namespace.name
+              - from: resource_attribute
+                name: k8s.node.name
+          - sources:
+              - from: resource_attribute
+                name: k8s.pod.ip
           - sources:
               - from: resource_attribute
                 name: k8s.pod.name
+              - from: resource_attribute
+                name: k8s.namespace.name
+          - sources:
+              - from: connection
         extract:
+          labels:
+            - tag_name: service.name
+              key: app.kubernetes.io/name
+              from: pod
+            - tag_name: service.name
+              key: k8s-app
+              from: pod
+            - tag_name: k8s.app.instance
+              key: app.kubernetes.io/instance
+              from: pod
+            - tag_name: service.version
+              key: app.kubernetes.io/version
+              from: pod
+            - tag_name: k8s.app.component
+              key: app.kubernetes.io/component
+              from: pod
           metadata:
-            - k8s.cluster.uid
```

Comment:
don't we still want that?

Reply:
It's there, it's just on line 717 now.

I think alpha-ordering these would probably make the most sense, but for now having the `k8sattributes:` config match up exactly between the different collectors is probably the best